HLTrigger/JetMET/src/HLTMETCleanerUsingJetID.cc

```diff
@@ -97,10 +93,10 @@ void HLTMETCleanerUsingJetID::produce(edm::Event& iEvent, const edm::EventSetup&
         double mex_diff = mex_goodJets - mex_jets;
         double mey_diff = mey_goodJets - mey_jets;
         //double sumet_diff = sumet_goodJets - sumet_jets;  // cannot set sumet...
-        reco::Candidate::LorentzVector p4_diff(mex_diff, mey_diff, 0, sqrt(mex_diff*mex_diff + mey_diff*mey_diff));
+        reco::Candidate::LorentzVector p4_diff(met->front().px()+mex_diff, mey_diff+met->front().py(), 0, sqrt((met->front().px()+mex_diff)*(met->front().px() +mex_diff)+(met->front().py()+mey_diff)*(met->front().py() +mey_diff)));
 
-        reco::CaloMET cleanmet = met->front();
-        cleanmet.setP4(cleanmet.p4() + p4_diff);
+        reco::CaloMET cleanmet ;
+        cleanmet.setP4(p4_diff);
```

Comment:
@safarzad 
Please address the comments from @kirschen and submit the corrections to the branch of this PR.
[We'll then request jenkins automatic testing in any case, as the PR can not be integrated without.]